app-config.yaml

```diff
@@ -158,30 +163,18 @@ techdocs:
   publisher:
     type: "local"
 
-# This is for k8s cluster management
-# Currently, just using a minikube cluster
-# TODO: Install backstage in GKE then point to GKE cluster
-
 kubernetes:
   serviceLocatorMethod:
-    type: "multiTenant"
-  clusterLocationMethdos:
-    - type: "config"
+    type: multiTenant
+  clusterLocatorMethods:
+    - type: config
       clusters:
-        - url: http://127.0.0.1:9999 # found by running `kubectl cluster-info`
-          name: minikube # unique within backstage project
-          authProvider: "serviceAccount" # uses a k8s sa, can use google sa
-          skipTLSVerify: false # k8s client verifies the TLS certificate presented by the API
+        - url:  ${k8s_url}
+          name: ${k8s_cluster_name}
+          authProvider: serviceAccount
```

Comment:
A little nit picky but we should keep these as upper case like K8S_URL and K8S_CLUSTER_NAME

Reply:
as best practice it must be, but intentionally put lower case due to ease for writing.
apologies, extra work for you!